layouts/blog/single.html

```diff
@@ -40,22 +33,33 @@
         </div>
         <div class='entry-header'>
           <h1 class='entry-title'> {{ .Title }}</h1>
+          <span class="author-blog ">
+            {{ if .Params.author }}
+
+            <!-- <a title='Posts by {{ .Params.author | default "" }}' rel="author"> -->
+            Author: {{ .Params.author | default ""}}
+            <!-- </a> -->
+            {{ else }}
+            {{ .Params.author | default "" }}
+            {{ end }}
+          </span>
         </div>
+
         <div class="entry-container">
           <div class="entry-content">
             <article>
               {{ .Content }}
             </article>
           </div>
           
-      <div class='entry-meta-bottom'>
+      <!-- <div class='entry-meta-bottom'>
```

Comment:
I don't think we should have comments in code unless they serve some obvious function (giving additional information or you can turn them on and get a working feature).